src/sql/workbench/contrib/query/browser/keyboardQueryActions.ts

```diff
@@ -131,6 +133,76 @@ export class RunCurrentQueryKeyboardAction extends Action {
 	}
 }
 
+export class CopyQueryWithResultsKeyboardAction extends Action {
+	public static ID = 'copyQueryWithResultsKeyboardAction';
+	public static LABEL = nls.localize('copyQueryWithResultsKeyboardAction', "Copy Query With Results");
+
+	constructor(
+		id: string,
+		label: string,
+		@IEditorService private _editorService: IEditorService,
+		@IClipboardService private _clipboardService: IClipboardService,
+		@IQueryModelService protected readonly queryModelService: IQueryModelService,
+	) {
+		super(id, label);
+		this.enabled = true;
```

Comment:
I wouldn't expect this to be necessary - what happens if you remove it?

Reply:
I removed it initially but all other methods had it so wasn't sure if its required for some other settings I am not aware of. I can remove it.